shopfloor/exceptions.py

```diff
@@ -0,0 +1,6 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+
+class ConcurentWorkOnTransfer(Exception):
```

Comment:
Never created an odoo exception, shouldn't it inherit from `odoo.exceptions.Whatever` ?

Reply:
Inheriting from the built-in Python exception

Reply:
it depends, if is an exception to show to users while they are doing something wrong, inherit from UserError

Reply:
Here's seems just an internal exception